submit-api/src/submit_api/models/note.py

```diff
@@ -0,0 +1,24 @@
+"""Internal staff document model class.
+
+Manages the internal staff
+"""
+
+from __future__ import annotations
+
+from sqlalchemy import Column
+
+from .base_model import BaseModel
+from .db import db
+
+
+class Note(BaseModel):
+    """Definition of the submitted documents entity."""
+
+    __tablename__ = "internal_staff_documents"
+
+    id = Column(db.Integer, primary_key=True, autoincrement=True)
+    note = Column(db.String(255), nullable=False)
+    item_id = Column(db.Integer, db.ForeignKey("items.id"), nullable=False)
+    created_by = Column(db.String, db.ForeignKey("users.auth_guid"), nullable=True)
+    created_date = Column(db.DateTime, nullable=False)
```

Comment:
I think created_date is already inherited from BaseModel so you don't need to add it again